packages/protocol/lib/compatibility/categorizer.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  Change,
+  ChangeVisitor,
+  ContractKindChange, DeployedBytecodeChange, MethodAddedChange,
+  MethodMutabilityChange, MethodParametersChange, MethodRemovedChange,
+  MethodReturnChange, MethodVisibilityChange, NewContractChange,
+  VISIBILITY_EXTERNAL, VISIBILITY_PUBLIC
+} from '@celo/protocol/lib/compatibility/ast-code'
+
+/**
+ * Change type categories according to semantic versioning standards
```

Comment:
nit: Consider adding a comment as to why the storage version type isn't relevant here

Reply:
The `categorizer` file contains standard semantic versioning assumptions, where storage is not one of them. Maybe what needs more commenting is why the storage is there in `version`, but here I think it could be weird